openreview/arr/management/setup_reviewer_matching.py

```diff
@@ -356,9 +356,9 @@ def replace_edge(existing_edge=None, edge_inv=None, new_weight=None, submission_
                 edge_readers=[venue_id, senior_area_chairs_id, area_chairs_id, reviewer_id]
             )
 
-        if venue.get_reviewers_id(number=submission.number, submitted=True) not in previous_reviewers.members:
-            current_client.add_members_to_group(previous_reviewers, venue.get_reviewers_id(number=submission.number, submitted=True))
-            current_client.add_members_to_group(previous_parent_reviewers, venue.get_reviewers_id(number=submission.number, submitted=True))
+        if venue.get_reviewers_id(number=submission.number, submitted=wants_new_reviewers) not in previous_reviewers.members:
```

Comment:
Just to double check, this means that:
if wants_new_reviewers==True:
- we add the `Reviewers/Submitted` group to the previous `/Reviewers` and `/Reviewers/Submitted` groups

if wants_new_reviewers==False:
- we add the `/Reviewers` group to the previous `/Reviewers` and `/Reviewers/Submitted` groups

Is this the intended behavior?

Reply:
Yes that's correct!